app/helpers/merge_record.rb

```diff
@@ -0,0 +1,22 @@
+module MergeRecord
+
+  # make this an ActiveSupport::Concern ?
+
+  def self.merge(model, where_clauses, record)
```

Comment:
What happens when one of the param hashes contains invalid data and `update` fails?